bedrock/base/templates/macros.html

```diff
@@ -474,27 +474,3 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     </footer>
   </section>
 {%- endmacro %}
-
-
-{% macro google_custom_search(search_type, type_class, default_term, term_list) %}
```

Comment:
All I remember was being asked to remove the macro in a second commit later on for fear of some kind of temporary breakage on deployment. Perhaps this is not the same thing.